Cargo.toml

```diff
@@ -7,6 +7,6 @@ license = "Apache-2.0 OR BSD-3-Clause"
 
 [dependencies]
 vmm = { path = "src/vmm" }
-cli = { path = "src/cli" }
+cli = { path = "src/api" }
```

Comment:
```suggestion
api = { path = "src/api" }
```

Reply:
This implies that you also rename `package` to `api` in `api/Cargo.toml` and that you should change `cli` to `api` in `main.rs`. `Cargo.lock` will also need to be updated.